calculator/calculator.py

```diff
@@ -1,15 +1,151 @@
 import re
+import math
+from typing import Union
+from statistics import mean
 
 
-class Calculator:
+__TOKEN_SPECIFICATION = {
```

Comment:
There is no meaning for the double under here, we are not in class (classes feature that rename/hides the attribute).
If you want to hide functions/module variables/methods use one under.
Read: https://dbader.org/blog/meaning-of-underscores-in-python